### PR TITLE
add "powered by beryx.io" 

### PR DESCRIPTION
**Pull request by juliangruber**
see comments for screenshots

### CODE REVIEW

**Comment by bajtos** [comment]
> are you familiar with the styling system?

Not at all :(

**Comment by juliangruber** [comment]
Options:

Screenshot 2024-02-29 at 14 00 20
Screenshot 2024-02-29 at 14 00 30
Screenshot 2024-02-29 at 14 00 41

Wdyt?

File: renderer/src/typings.ts [hunk 1]
```diff
@@ -26,6 +26,7 @@ declare global {
         getStationWalletTransactionsHistory: () => Promise<(FILTransaction|FILTransactionProcessing)[]>;
         transferAllFundsToDestinationWallet: () => Promise<void>;
         browseTransactionTracker: (transactionHash: string) => void;
+        openBeryx: () => void;
```
Comment (bajtos) [inline renderer/src/typings.ts:29]: I find it surprising that `openBeryx()` is nested under `stationConfig`.

Can we move it one level up, alongside `openReleaseNotes()`?

**Review by bajtos — APPROVED** [review]
👌🏻

**Pull request merged**